docs/github.md

```diff
@@ -0,0 +1,46 @@
+# GitHub Actions params fetching
+
+For multiple of these actions, you need to get the column_id for parameters. Here are the instructions on how to retrieve them.
+
+## Projects (classic)
+
+### Find column id
+
+Copy the column link and retrieve the id from the end of the URL
+
+## Projects (V2)
+
+### Find project number
+
+The project number is available in the URL of your project as: https://github.com/orgs/SonarSource/projects/<NUMBER>
+### Find column ID
+
+#### Create personal access token
+
+Create a personal access token (classic) with permissions for:
+- project
+
+[docs](https://docs.github.com/en/enterprise-server@3.9/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+#### query
+
+## Find data for Projects V2
+
+### Find project number
+
+The project number is available in the URL of your project as: https://github.com/orgs/SonarSource/projects/<NUMBER>
+
+
+### Find the column ID
+
+To get the column ids, use the following command, replacing `YOUR_GITHUB_PERSONAL_ACCESS_TOKEN` and `NUMBER`:
+
+```bash
+curl -H 'Content-Type: application/json' -H "Authorization: bearer <YOUR_GITHUB_PERSONAL_ACCESS_TOKEN>" -X POST -d '{ "query": "query {  organization(login: \"SonarSource\") { projectV2(number: <NUMBER>) { field(name: \"Status\") { ... on ProjectV2SingleSelectField { columns: options { id name }}}}}}" }' https://api.github.com/graphql
```

Comment:
I've fixed the first header with double quotes `"`, I tested it on a colleague's Windows machine. Not sure you have the same setup.

Reply:
what is your cURL version?

Reply:
curl 8.0.1 (Windows) libcurl/8.0.1 Schannel WinIDN
Release-Date: 2023-03-20
Protocols: dict file ftp ftps http https imap imaps pop3 pop3s smtp smtps telnet tftp
Features: AsynchDNS HSTS HTTPS-proxy IDN IPv6 Kerberos Largefile NTLM SPNEGO SSL SSPI threadsafe Unicode UnixSockets

Reply:
The current version still doesn't work. Based on this https://mkyong.com/web/curl-post-json-data-on-windows/ you'll need to use double quotes about the POST json and escape the inner quotes.

Reply:
I'm a 400 `Problems parsing JSON` error when using double quotes on `-d` like this:

```curl
curl -H "Content-Type: application/json" -H "Authorization: bearer <YOUR_GITHUB_PERSONAL_ACCESS_TOKEN>" -X POST -d "{ \"query\": \"query {  organization(login: \\"SonarSource\\") { projectV2(number: <NUMBER>) { field(name: \\"Status\\") { ... on ProjectV2SingleSelectField { columns: options { id name }}}}}}\" }" https://api.github.com/graphql
```

I'm not sure how to escape the inner quotes for `SonarSource` or `Status`.

However, I've found the usage of `--data-raw` in [MS docs](https://learn.microsoft.com/en-us/azure/developer/javascript/how-to/with-web-app/graphql/static-web-app-graphql/graphql-basics#http-curl-request). Would you try it?

```curl
curl -H "Content-Type: application/json" -H "Authorization: bearer <YOUR_GITHUB_PERSONAL_ACCESS_TOKEN>" -X POST --data-raw '{ "query": "query {  organization(login: \"SonarSource\") { projectV2(number: <NUMBER>) { field(name: \"Status\") { ... on ProjectV2SingleSelectField { columns: options { id name }}}}}}" }' https://api.github.com/graphql
```

Reply:
--data-raw is not going to help, because it's the console that fails to parse it.

Let's just give up and split it into 2 commands. Making one for both platforms seems too difficult.

Reply:
Do you mean splitting into different commands for Windows and MacOS?